src/app/routing/routing.module.ts

```diff
@@ -22,8 +26,20 @@ const routes: Routes = [
         component: ViewCaseComponent
     },
     {
-        path: 'viewcase/:case_id/:section/:section_item_id',
+        path: 'viewcase/:case_id/casefile/:section_item_id',
```

Comment:
You could leave this route for now and simply place below the specified question routes for now